pom.xml

```diff
@@ -448,12 +448,22 @@
       </dependency>
       <dependency>
         <groupId>org.jboss.resteasy</groupId>
-        <artifactId>resteasy-guice</artifactId>
+        <artifactId>resteasy-client-api</artifactId>
+        <version>${resteasy.version}</version>
```

Comment:
Lets use the resteasy-bom

```
<dependency>
    <groupId>org.jboss.resteasy</groupId>
    <artifactId>resteasy-bom</artifactId>
    <version>4.5.3.Final</version>
    <type>pom</type>
</dependency>
```

Reply:
224d842a I still had to pull in `resteasy-core` separately as that was not part of the bom. As per the migration guide, https://docs.jboss.org/resteasy/docs/4.0.0.Final/userguide/html/Migration_from_older_versions.html it seems like we shouldn
t directly rely on classes from the resteasy-core module. But we rely on that for tests I think. Is that fine?

Reply:
Looking at the code, I don't see a way for us not to use resteasy-core. I would log a issue/pr to resteasy to have the dependencies that are not managed by the bom, to be.